### ISSUE TITLE
Books not downloading

### ISSUE THREAD

**Issue by klinkehoffen** [openaudible/openaudible#136]
I am not able to download from audible, only seeing a grey dot. I am running OA v1.5 on Win 10.

It was working just fine until sometime after the latest OA update was installed. It has worked on one file since, and now it does not appear to be working at all. 

My Actions menu is greyed out, as is my right-click menu. As you can see by the attached image showing two screen shots (both menus), the items I want to download are showing that they are in the download queue. However, in more than 12 hours they are STILL in the download queue, with nothing to show that anything is happening. I am connected to the internet. Everything else is operating as normal.

I have tried launching the Audible window within OA, and clicking the download button there, but this also doesn't work.

My software verification button is unchecked.

Any ideas would be gratefully received.
![OpenAudible download options greyed out](https://user-images.githubusercontent.com/25827960/57333908-4e935680-7172-11e9-8c33-9ab3d67a2b83.jpg)

**Comment by openaudible** [comment]
According to the images, the tracks are either downloading or in the download queue. Check the File:log. Be patient. You can get the status of every book in the "Task" book status. 

Next release 1.5.1 will have an new icon that shows when a book has been added to the download or convert queue, which might help make things more clear.

**Comment by klinkehoffen** [comment]
Hi OpenAudible, the files are NOT downloading. The download status has now been the same for two days. I have shut down and restarted twice in case there is something else happening which is preventing download, but doing so has made no difference. 

The files are not downloading. Nothing I can do allows me to download them. The options are greyed out. This is quite frustrating. 

This call should not have been closed as the issue has not been resolved.

**Comment by klinkehoffen** [comment]
OK. Somehow solved this myself, though I am not sure what it was that worked. 
I deleted both the download files. I restarted PC then OpenAudible. Tried downloading again, but from within the Audible web window. Same problem. 
Deleted both the download files again. 
Restarted PC then OpenAudible. Sent a bug report from within OpenAudible. Tried downloading again, but using Crtl & D. Same problem. Deleted both the download files again. 
Restarted PC then OpenAudible. Changed the download folder locations. 
Restarted PC then OpenAudible. Tried downloading again, using Ctrl & D, while having my Audible account open in Chrome as well. Suddenly had an un-greyed out "Download" option on my right-mouse menu. The files downloaded, and converted fine.

I have no idea what of all of that did the trick.
